modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/CacheInfo.java

```diff
@@ -337,14 +342,15 @@ public Object name(VisorViewCacheCmd cmd) {
                 map.put("atomicity", getAtomicityMode());
                 map.put("backups", getBackupsCnt());
                 map.put("affCls", getAffinityClsName());
+                map.put("offHeapCnt", getOffHeapPrimaryEntriesCnt());
         }
 
         return map;
     }
 
     /** {@inheritDoc} */
     @Override public byte getProtocolVersion() {
-        return V2;
+        return V5;
```

Comment:
we already discuss it - why you need V5 here ? this is not global but local ver of current implementation.